mantidproject/mantid
# Expose FABADA minimizer options in ConvFit

FABADA options should be accessible from `Indirect Data Analysis` -> `ConvFit`.

**To test:**

- Open `Indirect Data Analysis` interface and go to `ConvFit`.
- Check `Use FABADA`, there should be a new checkable option called `Advanced`. When this option is checked, all of the FABADA options should be visible.
- Ensure that all the FABADA minimizer options (except output workspace names) are visible in the interface (you can check the minimizer properties [here](https://github.com/mantidproject/mantid/blob/master/Framework/CurveFitting/src/FuncMinimizers/FABADAMinimizer.cpp#L60-L111))
- Check the minimizer: load `irs26173_graphite002_red` as `Sample`, `irs26173_graphite002_res` as `Resolution`, select `One Lorentzian`, select `Use FABADA`, and set the number of iterations to `2e+06`. Hit `Fit Single Spectrum`. It will take a long time but the fit should finish successfully, although fit results may not be good.

Fixes #18745.

Release notes: https://github.com/mantidproject/mantid/pull/18757/commits/58350a7fdb2f7ad6ed46248cc89336ec73063dee

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
